services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -240,21 +238,21 @@ async def list_files(  # noqa C901
         return data
 
     async def get_file(self, user_id: UserID, file_id: StorageFileID) -> FileMetaData:
-        async with self.engine.acquire() as conn, conn.begin():
```

Comment:
Q: why do you remove `conn.begin()` I thought it was necessary to start the transaction.

Reply:
you are right, and I am wrong. I will double-check I did not remove too many